java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java

```diff
@@ -95,7 +96,7 @@ public void validateAllSupportedCiphers() throws IOException, DecoderException {
     /* Test Utility methods used in all unit-tests. */
 
     private void validateEncryptedFileWithHeaderValues(final SupportedCipherDetails cipherDetails) throws IOException, DecoderException {
-        final String cipherName = cipherDetails.getCipherId().replaceAll("/","-");
+        final String cipherName = cipherDetails.getCipherId().replaceAll(SEPARATOR,"-");
```

Comment:
Although the forward slash in this case is the same as `SEPARATOR`, they are being used differently. They are separating different content. I would undo this change.

Reply:
Noted.